reloj-checador/config/routes.rb

```diff
@@ -1,15 +1,15 @@
 Rails.application.routes.draw do
-
+  get "checkins/index"
```

Comment:
the route looks weird, what if you just use:
```ruby
get '/checkin"' to: 'checkins#index'
```

Reply:
Change maded now is get '/checkin', to: 'checkins#index'